<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueueWindow</class>
 <widget class="QMainWindow" name="QueueWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QListWidget" name="list"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart"/>
   <addaction name="actionPause"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionInfo"/>
   <addaction name="actionOpen_folder"/>
   <addaction name="actionSend_to_top"/>
   <addaction name="actionSend_to_bottom"/>
   <addaction name="actionRemove"/>
  </widget>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/play.svg</normaloff>:/icons/feather/play.svg</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/pause.svg</normaloff>:/icons/feather/pause.svg</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/stop.svg</normaloff>:/icons/feather/stop.svg</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionSend_to_top">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/chevrons-up.svg</normaloff>:/icons/feather/chevrons-up.svg</iconset>
   </property>
   <property name="text">
    <string>Send to top</string>
   </property>
  </action>
  <action name="actionSend_to_bottom">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/chevrons-down.svg</normaloff>:/icons/feather/chevrons-down.svg</iconset>
   </property>
   <property name="text">
    <string>Send to bottom</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/trash-2.svg</normaloff>:/icons/feather/trash-2.svg</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionOpen_folder">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/folder.svg</normaloff>:/icons/feather/folder.svg</iconset>
   </property>
   <property name="text">
    <string>Open folder</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/info.svg</normaloff>:/icons/feather/info.svg</iconset>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
